app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="container mb-3">
+  <div class="expense-header">
+     <h1> <%= link_to user_groups_path do %>
+        <i class="fa fa-arrow-circle-left" aria-hidden="true" style="color: #3778c2"> </i>
+        <% end %>
+     </h1>
+  </div>
+  <h3 class="d-flex justify-content-center align-items-center expense">TRANSACTONS</h3>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix `back` button's location and give style to details(`transactions`) part as [Behance](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) and also change design to `Add Transaction` button to [correct one](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) 🙏🏼 `[MOBILE VERSION]`

![image](https://github.com/YoutLone/Budget_App/assets/63915024/142c8d14-021c-46b3-a103-f2d4437b2fee)